src/integrationTest/resources/scripts/insert_case_event_messages_for_recived_messages_test.sql

```diff
@@ -0,0 +1,27 @@
+TRUNCATE TABLE public.wa_case_event_messages;
```

Comment:
This file seems to have a typo?

Reply:
THank you Ravi. I made necessary changes.